wormhole-connect/src/utils/wallet/evm.ts

```diff
@@ -96,29 +97,55 @@ export async function signAndSendTransaction(
   const signer = await (w as any).getSigner();
```

Comment:
`w` should always be an `EVMWallet` so we could create a `const w as EVMWallet` for better type checking

Reply:
<img width="1040" alt="image" src="https://github.com/user-attachments/assets/a769de8d-57f5-499a-8dc0-0946bfedca9c" />